src/store/modules/layers.store.js

```diff
@@ -165,10 +171,20 @@ const getters = {
      * Retrieves active layer(s) by ID
      *
      * @param {string} layerId ID of the layer(s) to retrieve
+     * @param {string} isExternal If the layer must be external, not, or both
+     * @param {string} baseUrl Base URL of the layer(s) to retrieve
      * @returns {[AbstractLayer]} All active layers matching the ID
```

Comment:
it could be a good idea to update the comment, as the method has changed a bit, even if the goal is clear.